.vscode/settings.json

```diff
@@ -1,5 +1,10 @@
 {
-  "python.testing.pytestArgs": ["test"],
+  "python.testing.pytestArgs": [
+    "test"
+  ],
   "python.testing.unittestEnabled": false,
-  "python.testing.pytestEnabled": true
-}
+  "python.testing.pytestEnabled": true,
+  "python.terminal.activateEnvironment": true,
+  "python.terminal.activateEnvInCurrentTerminal": true,
+  "python.defaultInterpreterPath": ".venv/bin/python",
```

Comment:
Makes sure the environment created inside the devcontainer is set in the VSCode so the developer doesn't need to remember...